src/test/java/model/GenerateLaddersTest.java

```diff
@@ -0,0 +1,16 @@
+package model;
+
+import org.junit.jupiter.api.Test;
+
+class GenerateLaddersTest {
+
+    @Test
+    void 사다리_생성_테스트() {
+        // given
+        GenerateLadders generateLadders = new GenerateLadders();
+        // when
+        generateLadders.generate(3,4);
+        // then
+    }
```

Comment:
1. 테스트가 잘못된 것 같습니다. 검증부가 없습니다!

2. 
```suggestion
    void 사다리_생성_테스트() {
        // given
        GenerateLadders generateLadders = new GenerateLadders();
        
        // when
        generateLadders.generate(3,4);
        
        // then
    }
```